translations/remote-schemas.kannada.md

```diff
@@ -0,0 +1,83 @@
+
+## ನಾವೀಗ ಆರಂಭಿಸೋಣ
+
+ರಿಮೋಟ್ ಸ್ಕೀಮಾವನ್ನು ಪ್ರಯತ್ನಿಸಲು ಹೀರೋಕು ವೇಗವಾದ ಮಾರ್ಗವಾಗಿದೆ.
+
+1. ಉಚಿತ ಪೋಸ್ಟ್‌ಗ್ರೆಸ್‌ನೊಂದಿಗೆ ಹೀರೋಕು ನಲ್ಲಿ ಎಂಜಿನ್ ಅನ್ನು ನಿಯೋಜಿಸಲು, ಈ ಕೆಳಗಿನ ಬಟನ್ ಅನ್ನು ಕ್ಲಿಕ್ ಮಾಡಿ:
+
+   [![Deploy to Heroku](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/hasura/graphql-engine-heroku)
+
+2. ಹಸುರಾ ಕನ್ಸೋಲ್ ತೆರೆಯಿರಿ
+
+   ನಿರ್ವಾಹಕ ಕನ್ಸೋಲ್ ತೆರೆಯಲು `https: // <app-name> .herokuapp.com` (_ ನಿಮ್ಮ ಅಪ್ಲಿಕೇಶನ್‌ನ ಹೆಸರಿನೊಂದಿಗೆ _ <app-name \> ಬದಲಾವಣೆ) ಗೆ ಹೋಗಿ.
+
+3. ನಿಮ್ಮ ಮೊದಲ ರಿಮೋಟ್ ಸ್ಕೀಮಾವನ್ನು ವಿಲೀನಗೊಳಿಸಿ ಮತ್ತು ಅದನ್ನು ಪ್ರಶ್ನಿಸಿ
+
+   ನಿರ್ವಾಹಕ ಕನ್ಸೋಲ್‌ನಲ್ಲಿ `Remote Schemas` ಟ್ಯಾಬ್ ತೆರೆಯಿರಿ ಮತ್ತು ` Add` ಬಟನ್ ಮೇಲೆ ಕ್ಲಿಕ್ ಮಾಡಿ. ಕೆಳಗಿನ ವಿವರಗಳನ್ನು ಭರ್ತಿ ಮಾಡಿ:
+
+   - Remote Schema ಹೆಸರು: `countries` (_ಈ ರಿಮೋಟ್ ಸ್ಕೀಮಾಗೆ ಅಲಿಯಾಸ್ _).
+   - ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಸರ್ವರ್ URL: `https://countries.trevorblades.com/` (_एಈ ವೈಶಿಷ್ಟ್ಯವನ್ನು ತ್ವರಿತವಾಗಿ ಪರೀಕ್ಷಿಸಲು ನಾವು ಬಳಸುವ ಸಾರ್ವಜನಿಕ ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಎಪಿಐ;[@trevorblades](https://github.com/trevorblades) ನಿರ್ವಹಿಸಿದವರು._
+   - ಉಳಿದ ಕಾನ್ಫಿಗರೇಶನ್ ಸೆಟ್ಟಿಂಗ್‌ಗಳನ್ನು ನಿರ್ಲಕ್ಷಿಸಿ ಮತ್ತು `Add Remote Schema` ಬಟನ್ ಮೇಲೆ ಕ್ಲಿಕ್ ಮಾಡಿ.
+
+   ``GraphiQL` ಟ್ಯಾಬ್ ಮಾಡಿ ಮತ್ತು ಈ ಕೆಳಗಿನ ಪ್ರಶ್ನೆಯನ್ನು ಚಲಾಯಿಸಿ (ಎಡಭಾಗದಲ್ಲಿರುವ ಪ್ರಶ್ನೆ ವಿಂಡೋದಲ್ಲಿ _ ಅಂಟಿಸಿ ಮತ್ತು _ ▶️ _(play) ಬಟನ್ ಮೇಲೆ ಕ್ಲಿಕ್ ಮಾಡಿ _):
+
+   ```graphql
+   {
+     countries {
+       emoji
+       name
+       languages {
+         name
+         native
+       }
+     }
+   }
+   ```
+
+   ನೀವು `GraphiQL` ಇಂಟರ್ಫೇಸ್ನ ಮೇಲಿನ ಬಲ ಮೂಲೆಯಲ್ಲಿ `Docs` ಎಕ್ಸ್‌ಪ್ಲೋರರ್ ಅನ್ನು ಬಳಸಿಕೊಂಡು ರಿಮೋಟ್ ಸ್ಕೀಮಾದಿಂದ ಗ್ರಾಫ್‌ಕ್ಯೂಎಲ್ ಪ್ರಕಾರಗಳನ್ನು ಕಂಡುಹಿಡಿಯಬಹುದು.
+
+## ಬಾಯ್ಲರ್ ಪ್ಲೇಟ್ಗಳು
+
+ಜನಪ್ರಿಯ ಭಾಷೆಗಳು/ಫ್ರೇಮ್‌ವರ್ಕ್‌ಗಳಲ್ಲಿ ಕಸ್ಟಮ್ ಗ್ರಾಫ್‌ಕ್ಯೂಎಲ್ ಸರ್ವರ್‌ಗಳಿಗೆ ಬಾಯ್ಲರ್‌ಪ್ಲೇಟ್‌ಗಳು ಲಭ್ಯವಿದೆ.
+
+- [ನಿಯಮಿತ boilerplates](../community/boilerplates) ಇದನ್ನು ಎಲ್ಲಿ ಬೇಕಾದರೂ ನಿಯೋಜಿಸಬಹುದು.
+- [ಸರ್ವರ್ ರಹಿತ boilerplates](https://github.com/hasura/graphql-serverless) AWS ಲ್ಯಾಂಬ್ಡಾ, ಇತ್ಯಾದಿಗಳಂತಹ ಸರ್ವರ್‌ಲೆಸ್ ಪ್ಲಾಟ್‌ಫಾರ್ಮ್‌ಗಳಲ್ಲಿ ಇದನ್ನು ನಿಯೋಜಿಸಬಹುದು.
+
+ಹೆಚ್ಚಿನ ಭಾಷೆಗಳು, ಚೌಕಟ್ಟುಗಳು, ಸರ್ವರ್‌ಲೆಸ್ ಪ್ಲಾಟ್‌ಫಾರ್ಮ್‌ಗಳು ಇತ್ಯಾದಿಗಳಿಗಾಗಿ ಬಾಯ್ಲರ್‌ಪ್ಲೇಟ್‌ಗಳನ್ನು ಪುನರಾವರ್ತಿಸಲಾಗುತ್ತಿದೆ ಮತ್ತು ಸಮುದಾಯದ ಕೊಡುಗೆಗಳು ತುಂಬಾ ಸ್ವಾಗತಾರ್ಹ ಎಂಬುದನ್ನು ದಯವಿಟ್ಟು ಗಮನಿಸಿ.
+
+## ಎಚ್ಚರಿಕೆಗಳು
+
+**ಪ್ರಸ್ತುತ ಮಿತಿಗಳು**:
+
+- ಪ್ರಕಾರದ ಹೆಸರುಗಳು ಮತ್ತು ನೋಡ್ ಹೆಸರುಗಳು ಎಲ್ಲಾ ವಿಲೀನಗೊಂಡ ಸ್ಕೀಮಾಗಳಾದ್ಯಂತ ಅನನ್ಯವಾಗಿರಬೇಕು (ಕೇಸ್-ಸೆನ್ಸಿಟಿವ್ ಮ್ಯಾಚಿಂಗ್). ಮುಂದಿನ ಕೆಲವು ಪುನರಾವರ್ತನೆಗಳಲ್ಲಿ, ನಿಖರವಾದ ಹೆಸರುಗಳು ಮತ್ತು ರಚನೆಗಳೊಂದಿಗೆ ವಿಲೀನ ವಿಧಾನಗಳಿಗೆ ಬೆಂಬಲ ಲಭ್ಯವಿರುತ್ತದೆ.
+- ವಿಭಿನ್ನ ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಸರ್ವರ್‌ಗಳಿಂದ ನೋಡ್‌ಗಳನ್ನು ಒಂದೇ ಪ್ರಶ್ನೆ/ಮ್ಯುಟೇಶನ್‌ನಲ್ಲಿ ಬಳಸಲಾಗುವುದಿಲ್ಲ. ಎಲ್ಲಾ ಉನ್ನತ ಮಟ್ಟದ ನೋಡ್‌ಗಳು ಒಂದೇ ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಸರ್ವರ್‌ನಿಂದ ಇರಬೇಕು.
+- ರಿಮೋಟ್ ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಸರ್ವರ್‌ನಲ್ಲಿನ ಚಂದಾದಾರಿಕೆಗಳು ಬೆಂಬಲಿತವಾಗಿಲ್ಲ. ರಿಮೋಟ್ ಗ್ರಾಫ್ಕ್ಯೂಎಲ್ ಸರ್ವರ್‌ಗಳಲ್ಲಿ ಚಂದಾದಾರಿಕೆಗಳನ್ನು ಬೆಂಬಲಿಸುವುದಿಲ್ಲ.
+
+ಈ ಮಿತಿಗಳನ್ನು ಮುಂದಿನ ಆವೃತ್ತಿಗಳಲ್ಲಿ ತಿಳಿಸಲಾಗುವುದು.
+
+## ದಸ್ತಾವೇಜನ್ನು
+
+ಪೂರ್ಣ ಓದಿ [ದಸ್ತಾವೇಜನ್ನು](https://hasura.io/docs/latest/graphql/core/remote-schemas/index.html).
+
+## ಅನುವಾದಿಸು
+
+ಈ ಡಾಕ್ಯುಮೆಂಟ್ ಈ ಕೆಳಗಿನ ಅನುವಾದಗಳಲ್ಲಿ ಲಭ್ಯವಿದೆ:
+
+- [Hindi :in:](translations/remote-schemas.hindi.md)
```

Comment:
French version can be linked here 
https://github.com/hasura/graphql-engine/blob/master/translations/remote-schemas.french.md